httpclient5/src/main/java/org/apache/hc/client5/http/config/RequestConfig.java

```diff
@@ -533,23 +533,25 @@ public Builder setConnectTimeout(final long connectTimeout, final TimeUnit timeU
         }
 
         /**
-         * Determines the timeout until arrival of a response from the opposite
-         * endpoint.
-         * <p>
+         * Determines the timeout until arrival of a response from the opposite endpoint.
```

Comment:
@LichKing-lee Please avoid formatting changes in the same change-set with the content changes